src/tox_travis/hooks.py

```diff
@@ -15,8 +16,8 @@
 from .after import travis_after
 
 
-@tox.hookimpl
-def tox_addoption(parser):
+@impl
+def tox_add_option(parser):
```

Comment:
Do we know when this was added to tox? Do we need to restrict the lower bound?

Reply:
Likely you should restrict to latest release, because that's also against what you're running the CI.

Reply:
See my comment below - this is not compatible with tox<4.